src/background.js

```diff
@@ -76,6 +75,7 @@ class StudyLifeCycleHandler {
    */
   enableFeature(studyInfo) {
     console.log("Enabling experiment", studyInfo);
+    browser.prefs.setStringPref("extensions.cookie-restrictions.wasEnabled", studyInfo.variation.name);
```

Comment:
uber-nit: could we call this `experiments.cookie-restrictions-breakage.varation`, which might be a little more specific?

Reply:
oh this is left over from the last one, should be removed. We won't be surveying after this one.